lib/ansible/modules/network/cloudengine/ce_sflow.py

```diff
@@ -127,6 +129,7 @@
               the switch automatically restores it to 65535.
               The value is an integer or 'all'.
         version_added: "2.10"
+        remove_version: "2.13"
```

Comment:
removed_in_version

Reply:
Also in argument_spec , i.e. " server_url=dict(type='str', required=True, removed_in_version="2.10")"